src/Pages/Home/Home.jsx

```diff
@@ -0,0 +1,58 @@
+import { Navbar } from "../../components/Navbar/Navbar";
+import fullSleeveTees from "../../assets/fullSleeveTees.jpg";
+import normalTees from "../../assets/normalTees.jpg";
+import pullOverHoodies from "../../assets/pullOverHoodies.jpg";
+import stencilHoodies from "../../assets/stencilHoodies.jpg";
+import bannerOptimized from "../../assets/bannerOptimized.jpg";
+import "../Home/home.css";
```

Comment:
you can store line no 2 to 6 in one file and export it like {fullSleeveTees,normalTees.. etc} 
for reference: https://www.taniarascia.com/react-architecture-directory-structure/
read componets part
and everything looks good

Reply:
Thanks, @mohammadkhizerkhan for the review. I have updated the changes suggested.